compiler/luci/import/src/CircleMetadataImporter.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_CIRCLE_METADATA_IMPORTER_H__
+#define __LUCI_CIRCLE_METADATA_IMPORTER_H__
+
+#include "luci/Import/CircleReader.h"
+
+#include <luci/Profile/CircleNodeOrigin.h>
+
+#include <map>
+#include <set>
+
+namespace luci
+{
+
+class CircleImportMetadata
+{
+public:
+  CircleImportMetadata() = delete;
+
+  CircleImportMetadata(const luci::CircleReader &reader);
+
+public:
+  /**
+   * @brief Create origin table using _source_table and _op_table in CircleImportMetadata
+   * @note  For creating origin table, both _op_table and _source_table should exist.
+   *        If one of them does not exist, empty table is returned.
+   */
+  const std::map<uint32_t, std::shared_ptr<CircleNodeOrigin>> origin_table(void);
```

Comment:
how about provide an alias for `const std::map<uint32_t, std::shared_ptr<CircleNodeOrigin>>` somewhere and use it?
`const std::map<uint32_t, std::shared_ptr<CircleNodeOrigin>>` is quite long...

Reply:
At first, as `const std::map<uint32_t, std::shared_ptr<CircleNodeOrigin>>` is only used in `CircleMetadataImporter.h` and `CircleMetadataImporter.cpp`, I didn't make alias for it :)

I will create an alias at this file!